src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunMinimalBWAMEMTest.java

```diff
@@ -0,0 +1,106 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Tests RunMinimalBWAMEM.
+ */
+public final class RunMinimalBWAMEMTest extends CommandLineProgramTest {
+
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "spark/sv/RunMinimalBWAMEM");
+    private static final File expectedSam = new File(TEST_DATA_DIR, "expected.sam");
+    private static final Path ref = Paths.get(b37_reference_20_21);
+    private static final Path bwaPath;
+    static {
+        if (System.getProperty("user.name").equalsIgnoreCase("shuang")) {
+            bwaPath = Paths.get("/usr/local/bin/bwa");
+        } else {
+            bwaPath = Paths.get("/home/travis/bwaApache2/bwa");
```

Comment:
this has a hardwired path. We need at least a way to override it on the commandline (can me via a parameter or system property)